modcc/printer/cprinter.cpp

```diff
@@ -81,6 +88,9 @@ std::string emit_cpp_source(const Module& module_, const std::string& ns, simd_s
         "#include <" << arb_header_prefix() << "backends/multicore/mechanism.hpp>\n"
         "#include <" << arb_header_prefix() << "math.hpp>\n";
 
+    with_profiling() &&
```

Comment:
Really? Abusing `&&` to form an if? That's very Perl-ish...
Just bite the bullet and add `if (with_profiling())`...

Reply:
It makes things line up better. Ultimately this is my fault for using the same construction elsewhere in the modcc printer source. (But I still think it looks prettier.)

Reply:
I wouldn't want to see this pattern used anywhere else, but it is fairly useful for these stream heavy operations in the modcc printers.

Reply:
Okay then -- maybe we put a comment that Sam is very naughty and lesser beings are not to follow his bad example.

Reply:
The pattern is used throughout the printers: so I don't think that there is a single sensible location to put the comment that will be seen by all unfortunate souls that wander into the printers.

It is a bit of an odd idiom, but used consistently throughout the printers, so I think readers might just have to "pick it up".